src/codeu/chat/common/Bot.java

```diff
@@ -0,0 +1,13 @@
+package codeu.chat.common;
+
+public abstract class Bot {
+
+    abstract String reactTo(String command, String[] args);
```

Comment:
We updated it into the body of the message/command and the sender of the command